pkg/cluster/px/px.go

```diff
@@ -977,68 +910,118 @@ func (ops *pxClusterOps) deleteAllPXComponents(clusterName string) error {
 		}
 	}
 
-	accounts := []string{
-		pxServiceAccountName,
-		lhServiceAccountName,
-		storkControllerServiceAccount,
-		storkSchedulerServiceAccount,
-		pvcControllerServiceAccount,
-		csiAccount,
+	clusterWideSecret := strings.Replace(clusterName+"_px_secret", "_", "-", -1)
```

Comment:
The goal for wipe is completely wipe the current cluster so that none of the remnants get in the way for the next install. If I leave this around, the next install with the same cluster name will end up having the same cluster wide secret. 
We warn about data loss in the prompt. So they shouldn't have any expectations of recovering data.